scripts/Synapse/analyzeLinkageAlgorithms.ipynb

```diff
@@ -135,16 +188,15 @@
       "metadata": {},
       "outputs": [],
       "source": [
-        "# MOUNT THE FILE SYSTEM SO WE CAN WRITE THE OUTPUTS TO FILES\n",
+        "import json\n",
+        "from notebookutils import mssparkutils\n",
+        "from pyspark.sql import SparkSession\n",
         "\n",
         "# Set paths\n",
         "STORAGE_ACCOUNT = \"$STORAGE_ACCOUNT\"\n",
         "LINKAGE_OUTPUTS_FILESYSTEM = f\"abfss://linkage-notebook-outputs@{STORAGE_ACCOUNT}.dfs.core.windows.net/\"\n",
         "BLOB_STORAGE_LINKED_SERVICE = \"$BLOB_STORAGE_LINKED_SERVICE\"\n",
         "\n",
-        "from notebookutils import mssparkutils\n",
-        "\n",
-        "# Functions for writing and reading results\n",
         "\"\"\"\n",
         "Function that writes the output of a linkage algorithm to a json file.\n",
         "\"\"\"\n",
```

Comment:
LOL exclusively because my natural habit is docstrings outside of functions and it's been so long since I looked at the SDK code base that I forgot we put them inside. So no, no intelligent reason 🤣